operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -5,14 +5,19 @@ is only relevant for booting compute nodes and can be ignored when working with
 
 This document describes the configuration of a Kubernetes NCN image. The same steps could be used to modify a Ceph NCN image.
 
-1. (`ncn-mw#`) Locate the NCN image to be modified.
+1. Identify the NCN image to be modified.
 
-    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by Kubernetes NCNs. However, the steps are the same for any Management NCN SquashFS image.
+    This example assumes that the administrator wants to modify the Kubernetes image that is currently in use by Kubernetes NCNs.
```

Comment:
It would be useful to provide a reason an admin might want to do this, give them an idea of what the use case is.